csvcubed/tests/unit/utils/qb/validation/test_observations.py

```diff
@@ -57,5 +63,63 @@ def test_find_sdmxa_obs_status_columns():
     }, obs_status_col_names
 
 
+def test_value_uri_template_is_present_in_existing_measure_dimention():
+    """
+    Ensuring that there is no validation error raised,
+    for a multi-measure column which re-uses existing measures when a csv_column_uri_template is provided.
+    """
+    qube = Cube(
+        metadata=CatalogMetadata("Some Qube"),
+        data=None,
+        columns=[
+            QbColumn("Some Dimension", NewQbDimension(label="Some Dimension")),
+            QbColumn(
+                "Values",
+                QbMultiMeasureObservationValue(
+                    unit=NewQbUnit("Some Unit"),
+                ),
+            ),
+            QbColumn(
+                "Measure", 
+                QbMultiMeasureDimension([
+                    ExistingQbMeasure("http://some-measure")
+                ]),
+                csv_column_uri_template="http://some-uri/{+measure}"
+            )
+        ],
+    )
+    errors = _validate_multi_measure_cube(qube, None)
+    assert len(errors) == 0, [e.message for e in errors]
+
+
+def test_value_uri_template_is_missing_in_existing_measure_dimention():
+    """
+    Ensuring that there is a validation error raised,
+    for a multi-measure column which re-uses existing measures when a csv_column_uri_template is provided.
+    """
+    qube = Cube(
+        metadata=CatalogMetadata("Some Qube"),
+        data=None,
+        columns=[
+            QbColumn("Some Dimension", NewQbDimension(label="Some Dimension")),
+            QbColumn(
+                "Values",
+                QbMultiMeasureObservationValue(
+                    unit=NewQbUnit("Some Unit"),
+                ),
+            ),
+            QbColumn(
+                "Measure", 
+                QbMultiMeasureDimension([
+                    ExistingQbMeasure("http://some-measure")
+                ]),
+            )
+        ],
+    )
+    errors = _validate_multi_measure_cube(qube, None)
+    assert len(errors) == 1, [e.message for e in errors]
+    assert 'CsvColumnUriTemplateMissingError' in str(errors)
```

Comment:
It's safer to do something like the below:

```python
error = errors[0]
assert isinstance(error, CsvColumnUriTemplateMissingError0
assert error.component == ExistingQbMeasure
```

This way, if (and when) we refactor the names of these things, the tests should automatically get refactored too. And we're checking the right attributes match.